ansible/roles/cluster/tasks/initialize.yml

```diff
@@ -25,10 +25,19 @@
     src: kubeadm-init.yml.j2
     dest: /etc/kubernetes/kubeadm-init.yml
 
-- name: initialize cluster
+- name: initialize cluster (with kube-proxy)
   command:
     cmd: kubeadm init --config /etc/kubernetes/kubeadm-init.yml --upload-certs
     creates: /etc/kubernetes/admin.conf
+  when:
+    - kube_proxy == "enabled"
+
+- name: initialize cluster (without kube-proxy)
+  command:
+    cmd: kubeadm init --skip-phases=addon/kube-proxy --config /etc/kubernetes/kubeadm-init.yml --upload-certs
```

Comment:
Is there a way to do this using the `kubeadm-init.yml` file? If not then this makes sense.

Reply:
Not that I found. Skip-phases was outside of the config file.